bug: time.Sleep should specify number of seconds to sleep for

```
tyk/rpc_analytics_purger.go:37:3: time.Sleep(sleep)
tyk-pump/pumps/elasticsearch.go:137:3: time.Sleep(5)
tyk-pump/pumps/influx.go:72:3: time.Sleep(5)
tyk-pump/pumps/mongo.go:125:3: time.Sleep(5)
tyk-pump/pumps/mongo_aggregate.go:333:3: time.Sleep(5)
tyk-pump/pumps/mongo_selective.go:104:3: time.Sleep(5)
tyk-pump/pumps/statsd.go:63:3: time.Sleep(5)
```

--- a/pumps/elasticsearch.go
+++ b/pumps/elasticsearch.go
@@ -134,7 +134,7 @@ func (e *ElasticsearchPump) connect() {
 		log.WithFields(logrus.Fields{
 			"prefix": elasticsearchPrefix,
 		}).Error("Elasticsearch connection failed: ", err)
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
 		e.connect()
 	}
 }

--- a/pumps/influx.go
+++ b/pumps/influx.go
@@ -69,7 +69,7 @@ func (i *InfluxPump) connect() client.Client {
 		log.WithFields(logrus.Fields{
 			"prefix": influxPrefix,
 		}).Error("Influx connection failed:", err)
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
 		i.connect()
 	}
 

--- a/pumps/mongo_aggregate.go
+++ b/pumps/mongo_aggregate.go
@@ -330,7 +330,7 @@ func (m *MongoAggregatePump) connect() {
 		log.WithFields(logrus.Fields{
 			"prefix": mongoAggregatePrefix,
 		}).Error("Mongo connection failed:", err)
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
 		m.connect()
 	}
 }

--- a/pumps/mongo_selective.go
+++ b/pumps/mongo_selective.go
@@ -101,7 +101,7 @@ func (m *MongoSelectivePump) connect() {
 		log.WithFields(logrus.Fields{
 			"prefix": mongoSelectivePrefix,
 		}).Error("Mongo connection failed:", err)
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
 		m.connect()
 	}
 }

--- a/pumps/segment_test.go
+++ b/pumps/segment_test.go
@@ -58,5 +58,5 @@ func TestSegmentPump(t *testing.T) {
 
 	go s.WriteData(tData)
 
-	time.Sleep(1000 * time.Millisecond)
+	time.Sleep(time.Second)
 }

--- a/pumps/statsd.go
+++ b/pumps/statsd.go
@@ -60,7 +60,7 @@ func (s *StatsdPump) connect() *statsd.StatsdClient {
 		log.WithFields(logrus.Fields{
 			"prefix": statsdPrefix,
 		}).Error("StatsD connection failed:", err)
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
 		s.connect()
 	}
 

